config/s3/s3api/bucket_operations.yaml

```diff
@@ -3,7 +3,7 @@ test_1:
   object_size:
     start: 0Kib
     end: 100Kib
-  result_duration: 1h
+  result_duration: 1m
```

Comment:
Revert this duration related changes before merging this PR in dev.

Reply:
Done